components/post_view/post_flag_icon/post_flag_icon.js

```diff
@@ -62,7 +63,7 @@ export default class PostFlagIcon extends React.PureComponent {
 
         return (
             <OverlayTrigger
```

Comment:
I noticed while working on the a11yController, that the `button` in this component (and possibly others, but I've only looked at this one so far) always loses focus when it is clicked as the `OverlayTrigger` is re-rendered to flip the tooltip from unselected to selected and back, causing the button to re-render as well and thereby lose focus, which messes up keyboard tabbing.

I'm not sure the best way to fix, but moving the `button` out to wrap the `OverlayTrigger` component fixed the problem while I was troubleshooting. Might be the case with other components as well.

Reply:
@deanwhillier I noticed that too, and that seems like a good solution (I have intentionally removed the tooltip triggers from this one). It may not work in places where we have a big button yet not a lot of content in it (the tooltip would then only open on the content its encapsulating).